wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,440 @@
+
+
+As part of this, an Actor's Rolldata is available as ["@attributes."](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) Useful examples:
+
+| @attribute                 | Description                   |
+| -------------------------- | ----------------------------- |
+| `@abilities.dex.mod`       | Actor's Dexterity Modifier    |
+| `@prof`                    | Actor's Proficiency Bonus     |
+| `@details.level`           | Actor's overall Level         |
+| `@classes.barbarian.levels` | Actor's Barbarian Class Level |
+
+  
+> Note that when using formulas in an Active Effect Value, the actor sheet display that corresponds to the changed value will not display the evaluated formula, but it will be applied when rolled.
+> E.g. When adding `+@abilities.cha.mod` to `system.bonuses.abilities.save` to simulate a Paladin's Aura of Protection, the actor sheet will not display that bonus applied to saving throws. The bonus will be present when the saving throw is rolled.
+
+To find out more about these, this post from Unsoluble in the [FoundryVTT Discord server](https://discord.gg/foundryvtt)'s #core-how-to channel can help:
+
+> To explore the data model in order to refer to inline properties like character levels, attributes, and other values, here are a few approaches:
+>
+> • Select a token, then open up the dev tools (F12 on Win; ⌥⌘I on Mac), and paste this into the Console:
+`console.log(canvas.tokens.controlled[0].actor.getRollData());`
+>
+> • Or: Install the "Autocomplete Inline Properties" module, to be able to just start typing in a supported field and have the available properties pop up (not all systems supported yet).
+>
+> • Or: Install this module (by pasting this manifest URL into the bottom of the Install Module window), enable it in your world, find and import its macro from the bundled Compendium, and run it on the selected token. `https://raw.githubusercontent.com/relick/FoundryVTT-Actor-Attribute-Lists/master/module.json`
+>
+> • Or: Right-click an actor in the sidebar and choose Export Data, which will get you a JSON file you can browse through. 
+
+
+## Commonly Desired Effect Examples
+
+### Abilities
+
+```
+system.abilities.[abbreviation].value
+                                bonuses.check
+                                        save
+```
+
+
+> <details>
+> <summary>Ability Abbreviations</summary>
+> 
+> | Ability      | Abbreviation |
+> | ------------ | ------------ |
+> | Strength     | `str`        |
+> | Dexterity    | `dex`        |
+> | Constitution | `con`        |
+> | Wisdom       | `wis`        |
+> | Intelligence | `int`        |
+> | Charisma     | `cha`        |
+> 
+> Source: `CONFIG.DND5E.abilities`
+>   
+> </details>
+
+
+#### Overriding an Ability Value
+E.g. an Item or potion that sets an ability score to a set value while in use
```

Comment:
An 'Upgrade' example would be nice here too. Like how Gauntlets of Ogre Power 'upgrade' Strength to 19, but will not lower it if the score is already higher.

Reply:
Unless we want to show that specific example, I think adding a table of the Change modes and their function at the top of the document might be more useful.